CellProfiler_pipelines/README.md

```diff
@@ -0,0 +1,288 @@
+These whole cells make up the object group called `Cells`.
+Any cells that are touching at least one edge of the image will be removed along with its respective nucleus. 
+This creates a new object group call `Nuclei`.
+
+- **IdentifyTeritaryObjects:** Identify cytoplasm by subtracting out the "smaller identfied objects" (`OrigNuclei`) from the "larger identified objects" (`Cells`). 
+`OrigNuclei` has to be used in this module over `Nuclei` due to an error downstream in the pipeline when using the latter object where it states "Cytoplasm is not in a 1:1 relationship with other objects".
+This means that the number of cytoplasm is not the same as the other objects, which would be wrong because there should be the same amount of objects (e.g the number of cells should match the number of cytoplasm and nuclei).
+
+Based on my understanding of CellProfiler, these modules identify the objects and create masks for each of them to use for feature extraction.
+
+### Feature Extraction
+
+Using the aforementioned object masks, each feature extraction module is applied to each image (from every channel).
+This process measures various features for cells, cytoplasm, and nuclei in each of the three channels.
+I followed the format of one of the CP pipelines from the [Broad Institute Platform Pipelines](https://github.com/broadinstitute/imaging-platform-pipelines/blob/master/cellpainting_UMUC9_20x_phenix_bin2/analysis_without_batchfile.cppipe).
+
+The following modules were used and their respective parameters (not including selecting images and objects, which should be all images and the obejcts `Nuclei`, `Cytoplasm`, and `Cells`):
+
+- **MeasureColocalization:**
+
+    - "Set threshold as percentage of maximum intensity for the images": `Integer value`
+    
+    The default is set to `15.0`. 
+    This sets a value to measure the correlation between intensities only for pixels that are above that threshold.
+
+    - "Select where to measure correlation": `Within objects`, `Across entire image`, or `Both`
+
+    The default is `Across entire image`, but I change it to `Within objects` as we are focused on the features from each of the single cells.
+
+    - "Run all metrics?": `Yes` or `No`
+
+    The default is `Yes`. When set to `No`, the user must choose the metrics they want. It is better to have all measurements run if you do not know what the various metrics are.
+
+    - " Method for Costes thresholding": `Faster`, `Fast`, or `Accurate`
+
+    The default is `Faster`. I use the default in this pipeline as it is what is used in the Broad Institute pipeline.
+
+- **MeasureGranularity:**
+
+    - "Measure within objects?" `Yes` or `No`
+
+    The default is `No`. It is changed to `Yes` as we are extracting features from the objects.
+
+    - "Subsampling factor for granularity measurement": `Integer value`
+
+    The default is set to `0.25`. I kept the default value and did not follow the Borad Institute because the documentation states:
```

Comment:
```suggestion
    The default is set to `0.25`. I kept the default value and did not follow the Broad Institute because the documentation states:
```